be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -426,6 +434,15 @@ Status VOlapTableSink::send(RuntimeState* state, vectorized::Block* input_block)
     if (findTabletMode == FindTabletMode::FIND_TABLET_EVERY_BATCH) {
         _partition_to_tablet_map.clear();
     }
+
+    //if pending bytes is more than 500M, wait
+    constexpr size_t MAX_PENDING_BYTES = 500 * 1024 * 1024;
+    if (get_pending_bytes() > MAX_PENDING_BYTES) {
+        while (get_pending_bytes() < MAX_PENDING_BYTES) {
```

Comment:
The if statement seems unnecessary, and the while statement's condition seems misjudged

Reply:
yes....